src/components/Common/Tab.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import is from 'styled-is'
+import UnstyledButton from './UnstyledButton'
+import { Subtitle } from '../Typography'
+
+const TabContainer = styled.li`
+  display: inline-block;
+
+  & + & {
+    margin-left: ${remcalc(36)};
+  }
+`
+
+const Text = styled(Subtitle)`
+  transition: opacity ${props => props.theme.animations.normal} ease-in-out;
+`
+
+const Button = styled(UnstyledButton)`
+  position: relative;
+
+  &:focus {
+    outline: ${remcalc(4)} solid ${props => props.theme.colors.outline};
+  }
+
+  &:active {
+    outline: none;
```

Comment:
Why?

Outline none should be added on focus and replaced with border and then nothing for focus

Reply:
No really sure what you mean....I'm doing the same as it's being done [in the modal](https://github.com/yldio/yld.io/blob/master/src/components/Training/Modal.js#L64)

Reply:
In the modal there is some change, in this case the background colour

You just removed all instances of a change for active :/

Reply:
you're right, I should'v added the same as the focus one 👍

Reply:
(or some other style, I'll check with antonas)